rotkehlchen/chain/evm/decoding/utils.py

```diff
@@ -50,7 +50,9 @@ def maybe_reshuffle_events(
 def bridge_prepare_data(
         tx_log: 'EvmTxReceiptLog',
         deposit_topics: Sequence[bytes],
+        # The chain of the current transaction
         source_chain: ChainID,
+        # The chain on the other side
         target_chain: ChainID,
```

Comment:
it took me a long time to get what `source` and `target` meant here. I was understanding them as `from` and `to` and didn't make sense when decoding on the side of the scroll chain. So I added some comments to clarify